.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
There is no need for this file in this PR. This as well as the eslint-related package installations should reflect in the eslint PR

Reply:
Remove this file entirely